### DISCUSSION TITLE
App A in deploy_map doesn't exist for job B

### DISCUSSION THREAD

**Discussion by pygabc1** [NVIDIA/NVFlare#1587] [Q&A] [answered]
Hi, when I run run_experiemnt_simulator.sh for xgboost (histogram-based), I confront the following error message.

> 2023-03-25 16:15:15,920 - SimulatorRunner - ERROR - Simulator setup error: App higgs_2_histogram_uniform_split_uniform_lr in deploy_map doesn't exist for job higgs_2_histogram_uniform_split_uniform_lr

![image](https://user-images.githubusercontent.com/31337772/227740022-f1ee63c6-850f-4520-b628-a413c759e61f.png)

As you see this screenshot, a folder workspaces/higgs_2_histogram_uniform_split_lr is generated when I run run_experiemnt_simulator.sh.
How to solve this issue?  I referred https://github.com/NVIDIA/NVFlare/pull/724 , but it does not solve my issue. 
It is window OS and 3.7 python, and Dev NVFlare version. Thanks.

**Comment by YuanTingHsieh** [comment, marked as answer]
Hi @pygabc1 thanks for the interest.

Thanks for the interest.
This issue will be fixed by: https://github.com/NVIDIA/NVFlare/pull/1596

Note that the dev branch is not the stable branch, it is development branch so sometimes it might break.
You are always welcome to use the latest release stable NVFlare, which is 2.2.5.
